src/api/delivery-needed/index.js

```diff
@@ -100,7 +101,7 @@ const makeFeature = async (r) => {
         need,
         slackPermalink: slackPermalink.ok ? slackPermalink.permalink : "",
         timestamp,
-        slackTs: metaJSON.slack_ts || "",
```

Comment:
We can also remove `timestamp` here! I believe it was a duplicate added awhile ago for the DaysOpenChip. Just in case you haven't already, might be worth manuallyy checking in the codebase if `slackTs` is used anywhere else. Unfortunately a lot of code is untested, so a manual search will be the best way to go about it,

Reply:
Already did :) It's only used as a parameter name [here](https://github.com/crownheightsaid/mutual-aid-app/blob/master/src/lib/airtable/tables/requests.js#L195)